FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -28,11 +24,13 @@
  */
 #define FBSnapshotVerifyViewWithOptions(view__, identifier__, suffixes__) \
 { \
+NSString *envReferenceImageDirectory = [NSProcessInfo processInfo].environment[@"FB_REFERENCE_IMAGE_DIR"]; \
```

Comment:
Yeah, I tried that out. The problem is that is is defined in the Application target. For my understanding this is too late. I get the following output when using `-D FB_REFERENCE_IMAGE_DIR="\"$(SOURCE_ROOT)/$(PROJECT_NAME)Tests/ReferenceImages\""` the output directory is

* __Objective-C__: `/Users/piet.brauer/Documents/Projekte/iOS/ios-snapshot-test-case/FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/ReferenceImages_64`
* __Swift__: `/Users/piet.brauer/Documents/Projekte/iOS/ios-snapshot-test-case/FBSnapshotTestCaseDemo/Pods/PodsTests/ReferenceImages_64`

I pushed to another branch for you to try it out: https://github.com/xing/ios-snapshot-test-case/tree/swift_old_macro